eng/service.proj

```diff
@@ -62,6 +62,49 @@
   <ItemGroup Condition="'$(ProjectListOverrideFile)' != '' ">
     <Track1ProjectsToRemove Include="$(MSBuildThisFileDirectory)..\sdk\*\Microsoft.*.Management.*\**\*.csproj;$(MSBuildThisFileDirectory)..\sdk\*mgmt*\**\*.csproj;$(MSBuildThisFileDirectory)..\sdk\*\Microsoft.Azure.*\**\*.csproj" />
     <ProjectReference Remove="@(Track1ProjectsToRemove)" />
+    <MgmtExcludePaths Include="$(MSBuildThisFileDirectory)..\sdk\*\Microsoft.*.Management.*\**\*.csproj;$(MSBuildThisFileDirectory)..\sdk\*mgmt*\**\*.csproj" />
```

Comment:
Does this mean none of these projects will be included in the PR pipeline? Also are these actual mgmt projects or are you just piggy backing on this include?

Reply:
> Does this mean none of these projects will be included in the PR pipeline? Also are these actual mgmt projects or are you just piggy backing on this include?

I'm piggy backing on the exclude, yeah.

Reply:
If we need these now I wonder if there is a better way to share them with the list above instead of duplicating.

Reply:
Will pick on re-using the item group.

Reply:
I moved the common pieces to an `itemgroup` above.